app/javascript/dashboard/routes/dashboard/helpcenter/components/PortalListItem.vue

```diff
@@ -95,15 +112,19 @@
                     'HELP_CENTER.PORTAL.PORTAL_SETTINGS.LIST_ITEM.PORTAL_CONFIG.ITEMS.SLUG'
                   )
                 }}</label>
-                <span class="text-block-title">{{ portal.slug }}</span>
+                <span class="text-block-title">
+                  {{ portal.slug }}
+                </span>
               </div>
               <div class="configuration-item">
                 <label>{{
                   $t(
                     'HELP_CENTER.PORTAL.PORTAL_SETTINGS.LIST_ITEM.PORTAL_CONFIG.ITEMS.TITLE'
                   )
                 }}</label>
-                <span class="text-block-title">{{ portal.page_title }}</span>
+                <span class="text-block-title">
+                  {{ portal.page_title }}
```

Comment:
We could stick to `camelCase` inside vue templates to maintain consistency, Thought it looks like an overkill we could move these keys as computed props.

@pranavrajs @muhsin-k @fayazara thoughts.